examples/jupyter/README.md

```diff
@@ -0,0 +1,113 @@
+<!--
+  Copyright (c) Microsoft Corporation
+  All rights reserved.
+
+  MIT License
+
+  Permission is hereby granted, free of charge, to any person obtaining a copy of this software and associated
+  documentation files (the "Software"), to deal in the Software without restriction, including without limitation
+  the rights to use, copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the Software, and
+  to permit persons to whom the Software is furnished to do so, subject to the following conditions:
+  The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
+
+  THE SOFTWARE IS PROVIDED *AS IS*, WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING
+  BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+  NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+  DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+-->
+
+
+# Jupyter on PAI
+
+This guide introduces how to run [Jupyter Notebook](http://jupyter.org/) on PAI.
+The following contents show some basic examples, other customized examples can be run similarly.
+
+
+## Contents
+
+1. [Basic environment](#basic-environment)
+2. [Advanced environment](#advanced-environment)
+3. [Jupyter Notebook example](#jupyter-notebook-example)
+
+
+## Basic environment
+
+First of all, PAI runs all jobs in Docker container.
+
+[Install Docker-CE](https://docs.docker.com/install/linux/docker-ce/ubuntu/) if you haven't. Register an account at public Docker registry [Docker Hub](https://hub.docker.com/) if you do not have a private Docker registry.
+
+We need to build a Jupyter Notebook image to run it on PAI, this can be done under following instructions:
```

Comment:
Actually, we don't need a dedicated 'Jupyter' image. Any image with Python installed can be used. We'd better assume the requirement to the user environment as small as possible, for the 'Basic' environment. So, in this section, I suggest we use an existing tensorflow image, and give an instruction about how to enable Jupyter using it on PAI.